yaptide/routes/celery_routes.py

```diff
@@ -22,94 +26,130 @@ class JobsDirect(Resource):
     @staticmethod
     @requires_auth(is_refresh=False)
     def post(user: UserModel):
-        """Method handling running shieldhit with server"""
+        """Submit simulation job to celery"""
         payload_dict: dict = request.get_json(force=True)
         if not payload_dict:
             return yaptide_response(message="No JSON in body", code=400)
+        
+        required_keys = {"sim_type", "ntasks", "input_type"}
 
-        if "sim_data" not in payload_dict:
+        if required_keys != required_keys.intersection(set(payload_dict.keys())):
             return error_validation_response()
```

Comment:
Add a message string mentioning which keys were expected and which in fact obtained

Reply:
I send back information about missing keys - more readable imo

Reply:
Thats good